src/util/accountUtils.ts

```diff
@@ -222,40 +205,14 @@ const getLinkNamespaceToAddressTransactionFee = (isMultisig :boolean,namespaceAd
   
 }
 
-const announceTransaction = (signedTransaction:SignedTransaction) => {
-  AppState.chainAPI.transactionAPI
-    .announce(signedTransaction)
-    .then(() => {
-        return true;
-    }, err => console.error(err)); 
-}
-
-const multiSigAnnouce = (aggregateTx:SignedTransaction, lockHashSigned:SignedTransaction) => {
-  // let hashLockAutoAnnounceSignedTx = new AutoAnnounceSignedTransaction(hashSigned);
-  // hashLockAutoAnnounceSignedTx.announceAtBlock = listenerState.currentBlock + 1;
-  let autoAnnounceSignedTx = new AutoAnnounceSignedTransaction(aggregateTx);
-  
-  autoAnnounceSignedTx.hashAnnounceBlock = new HashAnnounceBlock(lockHashSigned.hash);
-  autoAnnounceSignedTx.hashAnnounceBlock.annouceAfterBlockNum = 1;
-  autoAnnounceSignedTx.type = AnnounceType.BONDED;
-
-  AppState.chainAPI.transactionAPI.announce(lockHashSigned);
-
-  // ListenerStateUtils.addAutoAnnounceSignedTransaction(hashLockAutoAnnounceSignedTx);
-  ListenerStateUtils.addAutoAnnounceSignedTransaction(autoAnnounceSignedTx);
-
-  AppState.isPendingTxnAnnounce = true;
-}
-
-
 const linkNamespaceToAddress = (isMultisig :boolean,cosigners: [], multisigAccount: WalletAccount | OtherAccount, walletPassword: string, namespaceID: string, linkType: string, namespaceAddress: string) :SignedTransaction=> {
```

Comment:
should pass in only 1 selected cosigner to sign, if multisig

Reply:
and the data type is missing

Reply:
delegate and linkNamespaceToAddress doesnt have the UI for selecting cosigner, might need to add it

Reply:
as well as multisig, only transfer asset and namespace can select

Reply:
ya, will need to be able to select

Reply:
ok, will update accordingly, thanks